structure_tests/README.md

```diff
@@ -129,3 +129,18 @@ fileContentTests:
   expectedContents: ['.*httpredir\\.debian\\.org.*']
   excludedContents: ['.*gce_debian_mirror.*']
 ```
+
+### Running Structure Tests Outside of Container Build
+Structure tests can also be run outside of Cloud Container Build through a shell script, `ext_run.sh`. This allows the structure test framework to be used as normal presubmit tests in build systems like TravisCI. The only requirement to run is that the host machine has a working installation of Docker.
```

Comment:
This can come in another PR since it'll be hard to do before this goes in, but we should add a link on how to get the script?